image/templates/helm/stackrox-secured-cluster/templates/sensor-rbac.yaml

```diff
@@ -291,3 +291,35 @@ roleRef:
   kind: ClusterRole
   name: stackrox:review-tokens
   apiGroup: rbac.authorization.k8s.io
+---
+kind: ClusterRole
```

Comment:
No need for this extra CR/ CRB since sensor's service account already has [this permission](https://github.com/stackrox/stackrox/blob/ed197aedc54b94880ba60c9dc653f575b83f2cee/image/templates/helm/stackrox-secured-cluster/templates/sensor-rbac.yaml#L19-L36).